File: backend/components/metiers/administration_publique/utils/ai/core/ai_core.test.py
# ai_core.test.py
# Tests unitaires Python pour ai_core
from .ai_core import predict_ai

def test_predict_ai():
    assert 'Prédiction pour: Texte' in predict_ai('Texte')

def test_predict_ai_empty():
    assert predict_ai('') == '[AI-CORE] Entrée vide'

File: backend/components/metiers/administration_publique/utils/ai/core/ai_core.test.js
// ai_core.test.js
// Tests unitaires JS pour ai_core
const { predictAI } = require('./ai_core');

describe('predictAI', () => {
  it('retourne une prédiction IA pour une entrée', () => {
    expect(predictAI('Texte')).toMatch('Prédiction pour: Texte');
  });
  it('gère le cas vide', () => {
    expect(predictAI('')).toBe('[AI-CORE] Entrée vide');
  });
});
